content/en/docs/reference/glossary/rbac.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: RBAC (Role-Based Access Control)
+title: Role-Based Access Control (RBAC)
```

Comment:
One for SIG Auth to make the call on, I think

(might be easier omit that change from this PR, then propose it in a separate PR?)